ports/rp2/rp2_flash.c

```diff
@@ -259,3 +362,23 @@ mp_obj_t mp_vfs_rom_ioctl(size_t n_args, const mp_obj_t *args) {
     }
 }
 #endif
+
+// Modify the flash timing.  Ensure flash access is suspended while
+// the timings are altered.
+void rp2_flash_set_timing_for_freq(int clock_hz) {
+    if (multicore_lockout_victim_is_initialized(1 - get_core_num())) {
+        multicore_lockout_start_blocking();
+    }
+    uint32_t state = save_and_disable_interrupts();
+
+    rp2_flash_set_timing_internal(clock_hz);
+
+    restore_interrupts(state);
+    if (multicore_lockout_victim_is_initialized(1 - get_core_num())) {
+        multicore_lockout_end_blocking();
+    }
+}
+
+void rp2_flash_set_timing() {
```

Comment:
Please add a `void` in the argument list here.

Reply:
Done! Note to somehow lint for this?

Also adjusted both rp2_flash.h and rp2_psram.h to correct the include guards to `MICROPY_INCLUDED_RP2_RP2_FLASH_H` and `MICROPY_INCLUDED_RP2_RP2_PSRAM_H` since that accurately reflects their filename.

Reply:
> Done! Note to somehow lint for this?

We can do this by building (at least in CI) with -Wstrict-prototypes. I don't know whether MicroPython builds any third party sources that fail this warning, but I've made a note to find out.